specs/fulu/p2p-interface.md

```diff
@@ -304,13 +305,13 @@ During the deprecation transition period:
 **Protocol ID:** `/eth2/beacon_chain/req/data_column_sidecars_by_range/1/`
 
 The `<context-bytes>` field is calculated as
-`context = compute_fork_digest(fork_version, genesis_validators_root)`:
+`context = compute_fork_digest(genesis_validators_root, fork_epoch)`:
 
 <!-- eth2spec: skip -->
 
-| `fork_version`      | Chunk SSZ type           |
-| ------------------- | ------------------------ |
-| `FULU_FORK_VERSION` | `fulu.DataColumnSidecar` |
+| `fork_epoch`      | Chunk SSZ type           |
+| ----------------- | ------------------------ |
+| `FULU_FORK_EPOCH` | `fulu.DataColumnSidecar` |
```

Comment:
As you know we want the caller to always use the current slot (so the correct BPO entries are used). But looking at how `fork_version` was defined as 1 of a set of possible values in other specs, I think setting it to `FULU_FORK_EPOCH` here implies we are always supposed to use that literal value. Is that a correct parsing of spec conventions? If so I think we want to use the unfortunate `spec.get_current_epoch(state)`